src/Apps/Artwork/Components/ArtworkImageBrowser/ArtworkActionsSaveButton.tsx

```diff
@@ -23,7 +23,7 @@ const ArtworkActionsSaveButton: React.FC<ArtworkActionsSaveButtonProps> = ({
   })
 
   const { isAuction, isClosed } = artwork.sale ?? {}
-  const isOpenSale = isAuction && !isClosed
+  const isOpenOrUpcomingSale = isAuction && !isClosed
```

Comment:
Like @anandaroop noticed [here](https://github.com/artsy/eigen/pull/8962#discussion_r1254695682), `isOpenSale` poorly describes the real artwork status. The better wording would be "is open or upcoming", instead of just an "open" one.